hackathon_site/dashboard/frontend/src/components/inventory/InventoryFilter/InventoryFilter.module.scss

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
oops I forgot that I commited this, but it fixes and issue with the inventory button filter buttons being sticky